CMakeLists.txt

```
@@ -20,7 +20,9 @@ enable_testing()
 add_compile_options(-Wextra -Wall $<$<CONFIG:Debug>:-Werror> -std=gnu11 -D_GNU_SOURCE=1)
 add_executable(azure-nvme-id src/main.c)
 
-set(AZURE_NVME_ID_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/sbin")
+if(NOT DEFINED AZURE_NVME_ID_INSTALL_DIR)
```

cache path will allow this to be configurable:

```suggestion
set(AZURE_NVME_ID_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/sbin" CACHE PATH "azure-nvme-id installation directory")
```